src/main/java/com/openshift/restclient/ClientBuilder.java

```diff
@@ -183,7 +189,9 @@ public IClient build() {
 			dispatcher.setMaxRequests(maxRequests);
 			dispatcher.setMaxRequestsPerHost(maxRequestsPerHost);
 
-			OkHttpClient.Builder builder = new OkHttpClient.Builder()
+			OkHttpClient.Builder builder = null;
+			if (!this.sslCertCallbackWithDefaultHostnameVerifier) {
```

Comment:
duh ... for some reason I thought the `build()` was done as part of all that, but now I see it is on the next line.

will push an update momentarily